src/adapters/scene_direction.js

```diff
@@ -34,6 +34,9 @@ const createRouteLabel = (route, vehicle, { lngLat, anchor }) => {
     </div>
   `;
   element.className = `routeLabel routeLabel--${anchor}`;
+  if (vehicle === 'publicTransport') {
+    element.className += ' routeLabel--single-line';
```

Comment:
Sorry to add yet another comment on this bunch of code, but as I suggested in a previous review, `routeLabel--publicTransport` would be a better name.
`--single-line` is too descriptive of what it does and not of why we do it. It's better when we maintain the CSS to understand why.
Moreover, there are other planned developments on this feature where we enrich the content of these labels for public transport, so it will be already ready for other stuff than just hiding a line.

BTW, you could even simplify this by writing the transport mode as a class variant without any `if`.